nionswift_plugin/nion_experimental_4dtools/ComputationUI.py

```diff
@@ -59,15 +72,15 @@ def __display_item_changed(self, display_item):
                 for result in computation.results:
                     if result.specifier.get('uuid') == str(data_item.uuid):
                         computations_involved.append(computation)
-                if computation.source:
-                    if str(computation.source.uuid) == str(data_item.uuid):
-                        computations_involved.append(computation)
+                src = computation.get_input('src')
+                if src and str(src.uuid) == str(data_item.uuid):
+                    computations_involved.append(computation)
 
             self.__update_computation_ui(computations_involved)
             for _ in computations_involved:
                 self.__computation_updated_event_listeners.append(
                         self.document_controller._document_controller.document_model.computation_updated_event.listen(
-                                lambda data_item, new_computation: self.__update_computation_ui(computations_involved)))
+                                lambda data_item: self.__update_computation_ui(computations_involved)))
```

Comment:
If we make it `lamdba data_item, *args, **kwargs` then it will work with both 0.14 and 0.15 I think. I haven't tried it, though.

Reply:
Agreed, I will change and test it.